phpunit/class-wp-theme-config-resolver-test.php

```diff
@@ -1,15 +1,15 @@
 <?php
 
 /**
- * Test WP_Theme_JSON_Resolver class.
+ * Test WP_Theme_Config_Resolver class.
  *
  * @package Gutenberg
  */
 
-class WP_Theme_JSON_Resolver_Test extends WP_UnitTestCase {
+class WP_Theme_Config_Resolver_Test extends WP_UnitTestCase {
 
 	function test_presets_are_extracted() {
-		$actual = WP_Theme_JSON_Resolver::get_presets_to_translate();
+		$actual = WP_Theme_Config_Resolver::get_presets_to_translate();
```

Comment:
I think we can eventually remove this test in favor of the test I proposed #29828. It operates on a higher level and ensures that translations are applied. It would let us keep this method private since it is never used outside of the config resolver.